src/app/server/java/ChipFabricProvider-JNI.cpp

```diff
@@ -59,8 +58,6 @@ CHIP_ERROR AndroidChipFabricProviderJNI_OnLoad(JavaVM * jvm, void * reserved)
     env = JniReferences::GetInstance().GetEnvForCurrentThread();
     VerifyOrExit(env != NULL, err = CHIP_JNI_ERROR_NO_ENV);
 
-    chip::InitializeTracing();
```

Comment:
I think we still need this, otherwise custom trace has no initializer for the JNI code.

Reply:
this used to belong into src/trace. 

Currently it true that we did not provide tracing init in android and it has to be added. I would prefer to add it separately. I am not yet clear which OnLoad is supposed to do that (why did this file and `src/platform/android/AndroidChipPlatform-JNI.cpp` both have this?).

I would prefer to make android updates as a followup as they are independent.
Are you suggesting not removing `src/trace` yet until android is available in src/tracing ?

Reply:
We probably need some more generic startup hook for android, so that applications can choose to setup things, tracing among them (in which case they should register the perfetto tracing backend or just enable perfetto without a backend - in which case there would be no data logging).

Reply:
Enabled tracing using system perfetto for android by default.